Source/WebCore/rendering/RenderThemeIOS.mm

```diff
@@ -741,14 +741,21 @@ static inline bool canAdjustBorderRadiusForAppearance(StyleAppearance appearance
     if (!canAdjustBorderRadiusForAppearance(style.effectiveAppearance(), box) || style.backgroundLayers().hasImage())
         return;
 
-    if ((is<RenderButton>(box) || is<RenderMenuList>(box)) && box.height() >= largeButtonSize) {
-        auto largeButtonBorderRadius = std::min(box.width(), box.height()) * largeButtonBorderRadiusRatio;
+    auto isHorizontalWritingMode = style.isHorizontalWritingMode();
+    auto boxLogicalWidth = isHorizontalWritingMode ? box.width() : box.height();
+    auto boxLogicalHeight = isHorizontalWritingMode ? box.height() : box.width();
+
+    if ((is<RenderButton>(box) || is<RenderMenuList>(box)) && boxLogicalHeight >= largeButtonSize) {
+        auto largeButtonBorderRadius = std::min(boxLogicalWidth, boxLogicalHeight) * largeButtonBorderRadiusRatio;
```

Comment:
Taking the minimum of the two doesn't really require logical dimensions, but I guess that's fine.

Reply:
Makes me think we should only have these 2 variables:
```
auto boxLogicalHeight = box.logicalHeight();
auto minDimension = std::min(box.width(), box.height());
```
and reuse those across the method.

Reply:
Good idea.